lib/api/core/hotelClerk.js

```diff
@@ -180,10 +180,12 @@ function HotelClerk (kuzzle) {
     async.each(Object.keys(this.rooms), (roomId, callback) => {
       var room = this.rooms[roomId];
 
-      requestObjectCollection.index = room.index;
-      requestObjectCollection.collection = room.collection;
+      kuzzle.repositories.user.load(context.token.user)
+        .then(user => {
+          var requestObject = _.assign(requestObjectCollection, {index: room.index, collection: room.collection});
```

Comment:
Nitpicking: since `_.assign` mutates the object, it's simpler to get rid of the `requestObject` variable and simply use the updated `requestObjectCollection` object in subsequent calls:
```js
_.assign(requestObjectCollection, {index: room.index, collection: room.collection});
return user.isActionAllowed(requestObjectCollection, context, kuzzle);
```